packages/create-nx-workspace/bin/decorator.ts

```diff
@@ -0,0 +1,6 @@
+import * as chalk from 'chalk';
+
+export const yargsDecorator = {
+  'Options:': `${chalk.green('Options')}:`,
```

Comment:
Instead of using an arbitrary colour, what do you think of using `chalk.bold()` instead? That way we would respect any terminal theme options.